examples/nemo_ppo_sentiments.py

```diff
@@ -0,0 +1,93 @@
+# Generates positive movie reviews by tuning a pretrained model on IMDB dataset
+# with a sentiment reward function
+import json
+import os
+import sys
+from typing import List
+
+from datasets import load_dataset
+from transformers import DistilBertForSequenceClassification, pipeline
+
+import trlx
+from trlx.data.default_configs import (
+    TRLConfig,
+    default_nemo_1_3b_config,
+    default_nemo_20b_config,
+    default_ppo_config,
+)
+
+
+def get_positive_score(scores):
+    "Extract value associated with a positive sentiment from pipeline's output"
+    return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
+
+
+def main(hparams={}):
+    # Merge sweep config with default config if given
+    default_config = TRLConfig.update(default_ppo_config().to_dict(), hparams)
+
+    cfg_name = os.environ.get("NEMO_CONFIG", "1.3B")
+    if cfg_name == "1.3B":
+        nemo_config = default_nemo_1_3b_config()
+    elif cfg_name == "20B":
+        nemo_config = default_nemo_20b_config()
+    else:
+        raise ValueError(f"Unknown NEMO_CONFIG: {cfg_name}")
+
+    config = default_config.evolve(
+        train=dict(
+            seq_length=2048,
+            batch_size=32,
+            epochs=100,
+            eval_interval=64,
+            trainer="NeMoPPOTrainer",
+            trainer_kwargs=dict(
+                pretrained_model=f"/mnt/nvme/home/uwu/nemo-megatron-gpt-{cfg_name}/",
```

Comment:
maybe we should move these checkpoints to `/mnt/hdd`, otherwise they aren't accessible because of permissions

Reply:
moved